### ISSUE TITLE
when the moderator clicks on "moving on to the results" the question page should redirect to the page of the results.

### ISSUE THREAD

**Issue by pietrosperoni** [fairdemocracy/vilfredo-client#355]
Right now a person clicks on the question page, and everybody remains stuck in the page of the question. To go to the result page they have to go back to the home, and then from there click on the question again. This is crazy.

Once we move to the result it should all go directly to the result page for everybody. Reloading the page should send there. The moderator should go directly there. In short the previous page does not exist any longer. In fact we could as well put the results at the third phase of the question page. And you get the results while staying on that page.

P

**Comment by giovanisp** [comment]
2.0.1 is the Restart, we need urgent fixes there.

**Comment by giovanisp** [comment]
"In short the previous page does not exist any longer. "
Well it should exist, there is a button to go back to it. It is important to see the past votes and comments after the final results, specially if you missed voting.
Il 14/mag/2016 09:39, Pietro Speroni <notifications@github.com> ha scritto:Right now a person clicks on the question page, and everybody remains stuck in the page of the question. To go to the result page they have to go back to the home, and then from there click on the question again. This is crazy.

Once we move to the result it should all go directly to the result page for everybody. Reloading the page should send there. The moderator should go directly there. In short the previous page does not exist any longer. In fact we could as well put the results at the third phase of the question page. And you get the results while staying on that page.

P

**Comment by pietrosperoni** [comment]
@giovanisp 
Hmm, the info of what happened exists. But the page where you vote makes no sense to have it. Like having the voting cabinets still set up after the votes have happened. You keep the ballot boxes, but stored, and indexed. But the cabinet do not exist any longer.

**Comment by derekpa** [comment]
How about loading g the results at the top,of the question page?

**Comment by pietrosperoni** [comment]
yes, that would be fine. We will need to design the result page really well, as it will be some sort of take away message to the participants. But for now is a good starting point
